collections/Laud_lat/MS_Laud_Lat_21.xml

```diff
@@ -39,46 +39,159 @@
                </msIdentifier>
                <msContents>
                   <msItem class="#biblia" n="1" xml:id="MS_Laud_Lat_21-item1">
-                     <title key="work_13312" type="desc">Minor Prophets, with gloss</title>
-                     <bibl type="bible">
-                        <title>Bible, Minor Prophets</title>
-                     </bibl>
+                     <title key="work_13312" type="desc">Minor Prophets with glossa ordinaria</title>
```

Comment:
Is it technically the *Glossa ordinaria*? I have seen Lesley Smith and Alexander Andrée refer to earlier glossed biblical manuscripts as the *Glosa* (partially to recognize the degree of variety, but also since this is the term used by earlier commentators to cite such texts).

Reply:
I compared the text with Rusch's 1480 edition at several points and it did seem to be the ordinary gloss - perhaps this needs to be stated more explicitly though, with more of the text incipits given